azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -235,6 +240,20 @@ public KubernetesContainerizedImpl(final Props azkProps,
     }
     // Add all the job types that are readily available as part of azkaban base image.
     this.addIncludedJobTypes();
+
+    // Start the kubernetes watch if configured
+    // todo: this could benefit from a small api change in {@link ContainerizedDispatchManager}
+    //   which will also provide a suitable place for terminating the watch during server shutdown.
+    //   Will be discussed and included in a subsequent commit.
+    if (azkProps.getBoolean(ContainerizedDispatchManagerProperties.KUBERNETES_WATCH_ENABLED,
```

Comment:
Let's take this setup in different method.

Reply:
Will do.

Reply:
Done.